ISHPermissionKit/Requests/ISHPermissionRequestSpeechRecognition.m

```diff
@@ -0,0 +1,70 @@
+//
+//  ISHPermissionRequestSpeechRecognition.m
+//  ISHPermissionKit
+//
+//  Created by Hagi on 25/08/16.
+//  Copyright © 2016 iosphere GmbH. All rights reserved.
+//
+
+#import "ISHPermissionRequestSpeechRecognition.h"
+#ifdef NSFoundationVersionNumber_iOS_9_0
+
+#import "ISHPermissionRequest+Private.h"
+
+@import Speech;
+
+@implementation ISHPermissionRequestSpeechRecognition
+
+- (ISHPermissionState)permissionState {
+    return [self permissionStateForPhotosState:[SFSpeechRecognizer authorizationStatus]];
```

Comment:
Yap... sorry

It's all very repetitive, but since they're all different classes, I haven't found a proper way to reuse more code.